Aztec/Classes/Converter/CLinkedListToArrayConverter.swift

```diff
@@ -40,21 +40,21 @@ class CLinkedListToArrayConverter<ElementConverterType: Converter>: Converter {
     ///                     linked list into an element of the output array.
     ///     - next: the closure that will provide the next element in the list, given the current one.
     ///
-    init(elementConverter: ElementConverterType, next: (ElementConverterType.TypeIn) -> (TypeIn)) {
+    init(elementConverter: ElementConverterType, next: @escaping (ElementConverterType.TypeIn) -> (TypeIn)) {
         self.elementConverter = elementConverter
         self.next = next
     }
 
-    func convert(inputPtr: TypeIn) throws -> TypeOut {
+    func convert(_ inputPtr: TypeIn) throws -> TypeOut {
         var result = TypeOut()
-        var currentPtr = inputPtr
+        var nextPtr: UnsafeMutablePointer<ElementConverterType.TypeIn>? = inputPtr
```

Comment:
TypeIn is already declared as `UnsafeMutablePointer`. Do we need the double `UnsafeMutablePointer` for some reason?